src/MovieCard/MovieCard.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import './MovieCard.css';
 
-const MovieCard = ({ title, avgRating, photo, releaseDate }) => {
+const MovieCard = ({ title, avgRating, photo, releaseDate, history, movieId }) => {
   return (
-    <section className='movieCard'>
+    <section className='movieCard' onClick={() => {
+      history.push(`/${movieId}`)}}>
```

Comment:
This will be really helpful for providing access to the movie ID which will come in handy for our later iterations